templates/visuals/.api/v1.5.0/PowerBI-visuals.d.ts

```diff
@@ -956,73 +956,73 @@ declare module powerbi {
     /** Describes a data value type in the client type system. Can be used to get a concrete ValueType instance. */
     export interface ValueTypeDescriptor {
         // Simplified primitive types
-        readonly text?: boolean;
```

Comment:
There're two options:
* We should apply [this pull request](https://github.com/Microsoft/PowerBI-visuals-tools/pull/39) if we want to use TS 2.1.5
* Reject that pull request and update TS version in this pull request

What option do you prefer?